corona-warn-app/cwa-app-android
# Incompatibility warning card on main screen (EXPOSUREAPP-5464)(COMMUNITY)

Shows warning if device hardware is not supporting BLE scanning. Card opens FAQ link on click.

How to test:
Use some old phone running Android 6 or the emulator.
Eg. Nexus 5 displays the warning. Fairphone 2 is mentioned in the ticket.


Continuation of #2481 by @fynngodau 
Rebased and refactored.

Also see comments in #2481

---
Internal Tracking ID: [EXPOSUREAPP-5464](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5464)

![device-2021-04-26-093630](https://user-images.githubusercontent.com/49635654/116046011-f01f5580-a672-11eb-92b9-f73fb23949b1.png)
